sockeye/arguments.py

```diff
@@ -215,6 +230,28 @@ def add_model_parameters(params):
                               help="Add residual connections to stacked RNNs if --rnn-num-layers > 3. "
                                    "(see Wu ETAL'16). Default: %(default)s.")
 
+    # transformer arguments
+    model_params.add_argument('--transformer-model-size',
+                              type=int_greater_or_equal(1),
+                              default=512,
+                              help='Size of all layers and embeddings when using transformer. Default: %(default)s.')
+    model_params.add_argument('--transformer-num-layers',
```

Comment:
but we wouldn't be able to specify m layers for a transformer encoder, but only n layers for an RNN decoder. I'd like to keep this separate if possible.